weather_mv/loader_pipeline/bq.py

```diff
@@ -150,6 +150,8 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
 
     def __post_init__(self):
         """Initializes Sink by creating a BigQuery table based on user input."""
+        if self.zarr:
+            self.xarray_open_dataset_kwargs = self.zarr_kwargs
```

Comment:
Does this let us specify a range of years to open? If so, that's excellent and general purpose!

Reply:
Currently don't have that kind of support. But we can introduce a separate flag specific to `zarr` dataset to specify the time range. What do you think?

Reply:
Yes, that would be great. For now, I've seen that the pipeline runs for a quite a while with no apparent errors. I say: let's merge this CL for now, and implement the time range flags in a future PR.